pkg/storage/types.go

```diff
@@ -29,4 +27,6 @@ type StorageList struct {
 // StorageStatus is status of storage
 type StorageStatus struct {
 	Mounted bool `json:"mounted"`
+	// if path is empty, it indicates that the storage is not mounted in any component
```

Comment:
Do we still need `Mounted` than?

Reply:
I don't think of any case where it will be required since path will indicate the state. I leave the decision up to you. Should I remove it?

Reply:
Yes, if it doesn't make sense, please remove it.